iambic/plugins/v0_1_0/github/github.py

```diff
@@ -527,6 +533,57 @@ def handle_iambic_git_plan(
         raise e
 
 
+def handle_iambic_approve(
+    context: dict[str, Any],
+    github_client: github.Github,
+    templates_repo: github.Repo,
+    pull_request: PullRequest,
+    repo_name: str,
+    pull_number: str,
+    pull_request_branch_name: str,
+    repo_url: str,
+    proposed_changes_path: str = None,
+    comment_user_login: str = None,
+):
+    session_name = get_session_name(repo_name, pull_number)
+    os.environ["IAMBIC_SESSION_NAME"] = session_name
+
+    try:
+        repo_dir = get_lambda_repo_path()
+        _ = prepare_local_repo_for_new_commits(repo_url, repo_dir, "detect")
+        config_path = asyncio.run(resolve_config_template_path(repo_dir))
+        # it's important to load the config from main branch
+        # we want to guard against a requester directly changing the approver
+        # knowledge in the config of the iambic-template repository
+        main_config = asyncio.run(load_config(config_path))
+        github_main_config: GithubConfig = main_config.github
+        allowed_bot_approver = github_main_config.allowed_bot_approver
+
+        if (
+            comment_user_login is not None
+            and comment_user_login == allowed_bot_approver
+        ):
+            pull_request.create_review(
+                event="APPROVE", body=f"react to `approve` from `{comment_user_login}`"
+            )
+            return HandleIssueCommentReturnCode.APPROVED
+        else:
+            pull_request.create_issue_comment(
+                f"`{comment_user_login}` is not configured to approve PR via iambic"
+            )
+            return HandleIssueCommentReturnCode.UNDEFINED
+
+    except Exception as e:
+        captured_traceback = traceback.format_exc()
+        log.error("fault", exception=captured_traceback)
```

Comment:
I think you can run `log.error(..., exc_info=True)`  for the same effect, without needing to set captured_traceback